src/dxvk/dxvk_graphics.cpp

```diff
@@ -1384,6 +1410,8 @@ namespace dxvk {
     if (handle)
       m_fastPipelines.insert({ key, handle });
 
+    m_async = false;
```

Comment:
Use solution from DXVK-GPLALL 2.7-3, which I took from Britt Yazel DXVK GPLAsync 2.7 patch.
```suggestion
    m_async.store(false, std::memory_order_release);
```